js/plates-model/model-worker.js

```diff
@@ -105,6 +113,8 @@ onmessage = function modelWorkerMsgHandler (event) {
     if (storedModel) {
       self.m = model = Model.deserialize(storedModel)
     }
+  } else if (data.type === 'saveModel') {
+    postMessage({ type: 'saveModel', data: {savedModel: JSON.stringify(model.serialize())} })
```

Comment:
Perhaps I'd find it clearer if UI thered would receive "savedModel" or something like this, as "saveModel" sounds like a command request, not result. But it might be just me and it's not very important. 😉

Reply:
Naming here was definitely tough for me, since the UI thread is actually the one doing saving. I think your suggestion helps clarify things though, I'll still phrase things as commands to the model worker and results to the UI thread.